docs/fix/empty-wheelbarrows.rst

```diff
@@ -0,0 +1,33 @@
+fix/empty-wheelbarrows
+======================
+
+.. dfhack-tool::
+    :summary: Empties stuck items from wheelbarrows
+    :tags: fort bugfix items
+
+Empties all wheelbarrows which contain rocks that have become 'stuck' in them.
+
+This works around the issue encountered with :bug:`6074`, and should be run
+if you notice wheelbarrows lying around with rocks in them that aren't
+being used in a task.
+
+
+Usage
+-----
+
+``fix/empty-wheelbarrows [options]``
```

Comment:
```suggestion
::

    fix/empty-wheelbarrows [<options>]
```

Reply:
Are the angle brackets required? I've had the idea they were mainly used for required arguments as opposed to optional ones

Reply:
angle brackets indicate that the word between the brackets is a description. just `[options]` means "you can optionally (`[]`) add the literal word `options`". `[<options>]` means "you can optionally add various options here".

Reply:
Aighty, that's good to know. I've updated usage accordingly.